Dockerfile

```diff
@@ -1,6 +1,6 @@
 # 1. Generate licenses
 
-FROM node:16.13-alpine3.14 AS builder
+FROM node:18.16-alpine3.17 AS builder
```

Comment:
Could you please do the same change for alfresco-applications as I can see there is incorrect version too?

Reply:
yes raised a pr for same